docs/src/increase-observability/integrate-observability/blackfire.md

```diff
@@ -34,10 +34,13 @@ All customers can also subscribe to Blackfire separately.
 
 ### On a Grid plan or {{% names/dedicated-gen-3 %}} infrastructure 
 
-If you have the Platform.sh [Observability Suite](https://platform.sh/features/observability-suite/), 
-the [Blackfire automated integration](#automated-integration) is enabled on Grid environments by default. 
+If you have subscribed to a Grid plan complete with the Platform.sh [Observability Suite](https://platform.sh/features/observability-suite/),
+the [Blackfire automated integration](#automated-integration) is enabled on your environments by default. 
```

Comment:
:hammer_and_wrench: I am not sure about the `complete with` language. And maybe `Grid plan` isn't great because Grid is a kind of environment, not really a plan. It'd be nice if we had more clarity on terminology there :disappointed: 

```suggestion
If you have Grid environments on a plan with the Platform.sh [Observability Suite](https://platform.sh/features/observability-suite/),
the [Blackfire automated integration](#automated-integration) is enabled on your environments by default. 
```